eZ/Publish/Core/Search/Legacy/Content/Trash/Gateway/CriteriaConverter.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Search\Legacy\Content\Trash\Gateway;
+
+class CriteriaConverter extends \eZ\Publish\Core\Search\Legacy\Content\Common\Gateway\CriteriaConverter
```

Comment:
This class seems to be redundant but maybe I'm missed something. Please import `eZ\Publish\Core\Search\Legacy\Content\Common\Gateway\CriteriaConverter` :

```
use eZ\Publish\Core\Search\Legacy\Content\Common\Gateway\CriteriaConverter as BaseCriteriaConverter;

# ...

class CriteriaConverter extends BaseCriteriaConverter
{
    # ...
}    
```

Reply:
Earlier I understood that I have to add separate CriteriaConverter for new tagging, but if you want only a new tags without criteria converter I will change it